docs/books/admin_guide/09-backups.md

```diff
@@ -529,7 +523,7 @@ Specify a regular expression (*regex*) to extract the files matching the specifi
 For example, to extract all files with the extension `.conf`:
 
 ```bash
-tar xvf backup.tar --wildcards '*.conf'
+tar -xvf backup.tar --wildcards '*.conf'
 ```
```

Comment:
Actually, there are too many changes here to review in this way. In fact, If you feel strongly about using the options and including the minus sign, then I suggest you write your own tar guide. Your changes aren't wrong, but neither is the content you are changing. I know you've put a great deal of work and thought into this, but I'd suggest you write your own tar guide. If you want to copy some of this content first, so that you can create your own guide from the work you have already done, then I'll hold off deleting this PR for now.